e2e/various-suite/prometheus-config.spec.ts

```diff
@@ -13,33 +13,59 @@ describe('Prometheus config', () => {
     e2e.pages.AddDataSource.dataSourcePluginsV2(DATASOURCE_ID)
       .scrollIntoView()
       .should('be.visible') // prevents flakiness
-      .click();
-  });
-
-  it('should have a connection settings component', () => {
+      .click({ force: true });
+  });
+
+  it(`should have the following components: 
```

Comment:
nit: when I run prettier (on save), it removes this space: 

```suggestion
  it(`should have the following components:
```